README.md

```diff
@@ -44,12 +44,11 @@ Currently Frise only supports YAML files, but it may support JSON and other form
 
 ### Default values
 
-By defining directories where the files with default values can be found (in this example,
-[example/_defaults](example/_defaults)), Frise can handle its application internally on load time:
+By using the `$include` directive pointing to the files with default values can be found (in this example,
```

Comment:
Replace `with default values can be found` by `where default values can be found`.

Reply:
cb8215d